src/Php84/Resources/stubs/ReflectionConstant.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @author Daniel Scherzer <daniel.e.scherzer@gmail.com>
+ */
+if (\PHP_VERSION_ID < 80400) {
+    final class ReflectionConstant {
+
+        /**
+         * @var string
+         * @readonly
+         */
+        public $name;
+        
+        private $value;
+
+        /** @var bool */
+        private $deprecated;
+
+        /** @var bool */
+        private $persistent;
+        
+        public function __construct( string $name )
```

Comment:
```suggestion
        public function __construct(string $name)
```

We don't add spaces after opening and before closing parentheses. Can you please update this everywhere?

Reply:
Done here and everywhere else